docs/operational_guide/monitoring.md

```diff
@@ -0,0 +1,74 @@
+## Metrics
+
+TODO
+
+## Logs
+
+TODO
+
+## Tracing
+
+M3DB is integrated with [opentracing](https://opentracing.io/) to provide
+insight into query performance and errors.
+
+### Configuration
+Currently, only https://www.jaegertracing.io/ is supported as a backend.
+
+To enable it, set tracing.backend to "jaeger":
+
+```
+tracing:
+    backend: jaeger  # enables jaeger with default configs
+    jaeger:
+        # optional configuration for jaeger -- see
+        # https://github.com/jaegertracing/jaeger-client-go/blob/master/config/config.go#L37
+        # for options
+        ...
+```
+
+Jaeger can be run locally with docker as described in
+https://www.jaegertracing.io/docs/1.9/getting-started/.
+
+The default configuration will report traces via udp to localhost:6831;
+using the all-in-one jaeger container, they will be accessible at
+
+http://localhost:16686
+
+
+#### Alternative backends
+
+If you'd like additional backends, we'd love to support them!
+
+File an issue against M3 and we can work with you on how best to add
+the backend. The first time's going to be a little rough--opentracing
+unfortunately doesn't support Go plugins (yet--see
+https://github.com/opentracing/opentracing-go/issues/133), and `glide`'s
+update model means that adding dependencies directly will update
+*everything*, which isn't ideal for an isolated dependency change.
+These problems are all solvable though,
+and we'll work with you to make it happen!
+
+### Use cases
+
+Note: all URLs assume a local jaeger setup as described in Jaeger's
+[docs](https://www.jaegertracing.io/docs/1.9/getting-started/).
+
+
+#### Finding slow queries
+
+To find prom queries longer than <threshold>, filter for `minDuration >= <threshold>` on
+`operation="GET /api/v1/query_range"`.
+
+Sample query:
+http://localhost:16686/search?end=1548876672544000&limit=20&lookback=1h&maxDuration&minDuration=1ms&operation=GET%20%2Fapi%2Fv1%2Fquery_range&service=m3query&start=1548873072544000
```

Comment:
nit: may be nicer to wrap it in `\`\`` characters otherwise it'll be a real link :)

Reply:
Feature not a bug I think; if you're running jaeger locally, you should just be able to click and do the search.